src/devices/Tm1637/samples/Tm1637.sample.cs

```diff
@@ -52,7 +52,7 @@ public static void Main(string[] args)
             Thread.Sleep(3000);
             for (int i = 0; i < 6; i++)
             {
-                rawData[i] = (Character)Enum.Parse(typeof(Character), $"Char{i}");
+                rawData[i] = (Character)Enum.Parse(typeof(Character), $"Digit{i}");
```

Comment:
can you also update SegmentOrder so that it works correctly also on 4 segment? i.e.: ` tm1637.SegmentOrder = new byte[] { 2, 1, 0, 3, 5, 4 };`

Reply:
Actually, I've faced similar issue before fixing `Display(byte, byte)`.
`Display(ReadOnlySpan<Character>)` seems to be unstable or broken.
It sometime shows like [0 23], or [10 2], not [0123]. Some segment become blank. Or displays character on different segment...
And it is not because of `SegmentOrder` at least.

So I tried to fix `Display(byte, byte)` first to find the reason.
I think `Display(ReadOnlySpan<Character>)` has another problem...

Reply:
Ok, will merge this as is since without it I can't even start the sample. Please add anything you're fixing to the sample project for testing purposes if it's not being tested yet.

Reply:
Thank you for merging😄

And do you mean I have to add some code which using `Display(byte, byte)` to the sample project?

Reply:
yes, anything you find not working :smile:

Reply:
Thank you. Sample project updated (#910)